source/openapi3/split.yaml

```diff
@@ -815,6 +815,20 @@ info:
     check out our article on [what happens when an NPP Payment fails](https://help.zepto.money/en/articles/4405560-what-happens-when-an-npp-payment-fails)
     to learn more about what happens when an NPP Payment is unable to process.
 
+    ## NPP & DE Payment failures (channel switching)
+
+    ### Using failure codes
+
+    To simulate a transaction failure across both NPP and DE channels, create a Payment with an amount corresponding to the desired failure code.
```

Comment:
```suggestion
    To simulate a transaction failure across both NPP and DE channels, create a Payment with an amount corresponding to the desired failure code. Any Payment with an amount in between $4.00 and $4.99, inclusive, will cause the NPP leg of the payment will fail. The cent portion of the amount is then used to determine what failure code to fail the DE attempt with in the event that the Payment is channel switched.
```

Reply:
You probably have a better way of explaining this haha but think it's important to highlight that any $4.xx amount will fail via NPP